ref/app-architecture.md

```diff
@@ -0,0 +1,32 @@
+
+Index.php
+    Instantiate DB connection object
```

Comment:
This isn't readable markdown. Could you change the styling? I suggest this format:
```Index.php

- Instantiate DB connection object
- Pulls breed list (as array of strings) from DB using DB connection object
- If breed selection given:
    - Give CatHydrator the DB connection object and return the list of CatEntity objects
        Add main HTML for Top Cat page```